ChangeLog

```diff
@@ -120,6 +120,8 @@ Release date: tba
 
       Closes issue #911.
 
+	* Fix a false negative of 'useless-super-delegation', occurred when the method
+	  there are *decorators*.
```

Comment:
That check was added in the same release, so no point to have a changelog entry for that IMHO?